acceptance-tests/README.md

```diff
@@ -16,20 +16,20 @@ Next, in the navigation bar, click "Testing", and then "Create app test account"
 
 Now, navigate back to the "Testing" page, and click your test account. Take note of the Account ID in the URL (`https://app.hubspot.com/dashboard-library/ACCOUNT_ID`). Then [navigate to the following page and generate a personal access key](https://app.hubspot.com/portal-recommend/l?slug=personal-access-key) that is associated with your test account.
 
-Generate [a github token](https://github.com/settings/tokens/new) to test with.
+Generate [a github token](https://github.com/settings/tokens/new) to test with. This is required for the CLI commands that interact with the github api.
```

Comment:
I need to double check why this token is needed. From my experience so far it's not necessary.